packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -37,10 +37,16 @@ class CreateCommand extends FlutterCommand {
       help: 'Also add a flutter_driver dependency and generate a sample \'flutter drive\' test.'
     );
     argParser.addFlag(
-      'plugin',
+      'package',
```

Comment:
My main concern with having multiple flags is that then we have to handle the cases of them being specified together (--plugin --package), which gets confusing to explain (you can use _these_ together, but not _these_, etc), especially since we have other flags that _do_ work in combination.

`--debug`, `--profile`, and `--release` are an unfortunate set of values that we've grandfathered in because that's what people expect coming from other build systems, but I don't think that's a problem here.